src/Compilers/Core/Portable/Diagnostic/LocalizableString.cs

```diff
@@ -46,7 +46,7 @@ public string ToString(IFormatProvider? formatProvider)
             return FixedLocalizableString.Create(fixedResource);
         }
 
-        public sealed override string? ToString()
+        public sealed override string ToString()
```

Comment:
@dotnet/roslyn-compiler very simple change.  but this was pretty annoying downstream as this can't actually return null, but callers have to be resilient to it based on this annotation.

Reply:
Makes sense